src/ontology/mondo-edit.obo

```diff
@@ -160535,6 +160535,7 @@ intersection_of: MONDO:0020573 ! inherited disease susceptibility
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/3431 ! ERBB3
 intersection_of: predisposes_towards MONDO:0017858 ! acute erythroid leukemia
 relationship: excluded_subClassOf MONDO:0011118 {source="EFO:0000218", source="https://orcid.org/0000-0001-5208-3432"} ! bilineal acute myeloid leukemia
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/3431 ! ERBB3
```

Comment:
@sabrinatoro v2024-12-03/mondo.owl does not have a source for this gene association